lxgettext/lxgettext.py

```diff
@@ -0,0 +1,190 @@
+import argparse
+import datetime
+import os
+import re
+
+import polib
+
+COLOUR_GREEN = '\033[92m'
+COLOUR_END = '\033[0m'
+
+KEYWORD = "gettext"
+gettext_re = re.compile("""%s\(['"](.+?)['"]\)""" % KEYWORD)
```

Comment:
this won't catch `gettext("hi there :') hola if ya hear!")`
Not sure it's a problem though